biojava-core/src/main/java/org/biojava/nbio/core/sequence/template/AbstractSequence.java

```diff
@@ -323,6 +325,22 @@ public void setSequenceScore(Double sequenceScore) {
 	}
 
 	/**
+	 *
+	 * @return
+	 */
+	public List<AbstractReference> getReferences() {
```

Comment:
Just keep committing to the same branch. The Pull Request will automatically update with those commits.